.env

```diff
@@ -0,0 +1,4 @@
+MUMBAI_API_URL=""
+POLYGON_API_URL=""
+PRIVATE_KEY="0000000000000000000000000000000000000000000000000000000000000000"
+POLYSCAN_KEY=""
```

Comment:
`.env` are not supposed to be committed they are gitignored files to be used by each developer to put their access keys inside so they can be accessed during development....

This file mainly seems to serve the purpose of putting dummy values for testing inside... I don't think it's needed....

A better approach for this problem would be write a wrapper scripts for running the tests....something like

```bash
#!/bin/bash

# Dummy values for test
export MUMBAI_API_URL=""
export POLYGON_API_URL=""
export PRIVATE_KEY="0000000000000000000000000000000000000000000000000000000000000000"
export POLYSCAN_KEY=""

hardhat test
```

Putting these dummy values in a wrapper script would have the advantage of still actively requiring them for actual deployment

Reply:
I added it into the workflow and deleted .env file